editoast/src/models/layers/mod.rs

```diff
@@ -0,0 +1,5 @@
+mod geo_json_and_data;
+
+pub use geo_json_and_data::create_and_fill_mvt_tile;
+pub use geo_json_and_data::GeoJsonAndData;
+pub use geo_json_and_data::GeoPoint;
```

Comment:
```suggestion
pub mod geo_json_and_data;
```

wouldn't that be enough?